hack/kubeadminkubeconfig/get.go

```diff
@@ -64,8 +81,11 @@ func main() {
 		log.Fatalf("usage: %s resourceid\n", os.Args[0])
 	}
 
```

Comment:
Use wait.PollImmediateUntil(). also put interval times directly into the call so the reader does not have look elsewhere.

Reply:
OK - do we want to just retry indefinitely or add a timeout to it for a hard fail?